libs/basics/src/iterators/async_iterator_plus.ts

```diff
@@ -391,6 +391,10 @@ export class AsyncIteratorPlusImpl<T> implements AsyncIteratorPlus<T> {
     return set;
   }
 
+  async toString(separator = ''): Promise<string> {
```

Comment:
opt: Would `.join(separator)` naming work better semantically then `.toString(separator)`? That might be closer to the syntax people are used to when working with arrays.

Reply:
That's a fair point. As `toString` would otherwise have no useful implementation, what do you think of making it an alias for `join`?

Reply:
Ah, that works too!